pre_push.py

```diff
@@ -41,11 +47,14 @@ def main():
     finally:
         rmtree(tmp_dir)
 
-    return 0 if success else 1
+    curdir = path.abspath(path.join(__file__, ".."))
```

Comment:
While the tests run _fairly_ quickly, running the tests require having a development environment set up. The intent for this script was to quickly run static checks to catch quick to resolve issues before pushing. I'd still like there to be a way to accomplish that without also running tests. Perhaps a command line argument?

Reply:
I changed it to be command-line style

Reply:
> While the tests run _fairly_ quickly, running the tests require having a development environment set up. The intent for this script was to quickly run static checks to catch quick to resolve issues before pushing. I'd still like there to be a way to accomplish that without also running tests. Perhaps a command line argument?

```
usage: pre_push.py [-h] [-n] [-u] [-a]

Run static and/or unit-tests

optional arguments:
  -h, --help            show this help message and exit
  -n, --unstatic        Do not run static tests
                        (black/flake8/pydocstyle/sphinx-build)
  -u, --unit-tests, --unit
                        Run the unit tests
  -a, --all             Run all of the tests (static and unit). Overrides the
                        unstatic argument.
```

This is the solution. I understand having `-a` is redundant, but if I was new I would expect there to be an all option